third-party/nerdynick-elasticsearch-metrics-reporter/pom.xml

```diff
@@ -0,0 +1,255 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    
+    <groupId>org.elasticsearch</groupId>
+    <artifactId>metrics-elasticsearch-reporter</artifactId>
+    <version>6.0.0-SNAPSHOT-apache-james</version>
```

Comment:
We should not introduce this kind of SNAPSHOT dependencies as it makes releasing harder.

Reply:
BTW I fear that we will not be able to release this. Can't we change the groupId to `org.apache.james` ?